tests/circuit_breaking_test.py

```diff
@@ -0,0 +1,214 @@
+            default_test_servers = self.startTestServers()
+            alternate_test_servers = self.startTestServers(
+                server_runner=self.alternate_server_runner
+            )
+
+        with self.subTest("06_add_server_backends_to_backend_services"):
+            self.setupServerBackends()
+            # Add backend to alternative backend service
+            (
+                neg_name_alt,
+                neg_zones_alt,
+            ) = self.alternate_k8s_namespace.parse_service_neg_status(
+                self.alternate_server_runner.service_name, self.server_port
+            )
+            self.td.alternative_backend_service_add_neg_backends(
+                neg_name_alt, neg_zones_alt
+            )
+
+        with self.subTest("07_patch_backends_with_circuit_breakers"):
+            self.td.backend_service_patch_backends(
+                circuit_breaker={"maxRequests": _INITIAL_UNARY_MAX_REQUESTS}
+            )
+            self.td.alternative_backend_service_patch_backends(
+                circuit_breaker={"maxRequests": _INITIAL_EMPTY_MAX_REQUESTS}
+            )
+
+        test_client: _XdsTestClient
+        with self.subTest("08_start_test_client"):
+            test_client = self.startTestClient(
+                default_test_servers[0], rpc="UnaryCall,EmptyCall", qps=_QPS
+            )
+
+        with self.subTest("09_test_client_xds_config_exists"):
+            self.assertXdsConfigExists(test_client)
+
+        with self.subTest("10_test_server_received_rpcs_from_test_client"):
+            self.assertRpcsEventuallyGoToGivenServers(
+                test_client, default_test_servers + alternate_test_servers
+            )
+
+        with self.subTest("11_configure_client_with_keep_open"):
+            test_client.update_config.configure(
+                rpc_types=["UNARY_CALL", "EMPTY_CALL"],
+                metadata={
+                    ("UNARY_CALL", "rpc-behavior", "keep-open"),
+                    ("EMPTY_CALL", "rpc-behavior", "keep-open"),
+                },
+                timeout_sec=20,
+            )
+
+        with self.subTest("12_client_reaches_target_steady_state"):
+            self.assertClientEventuallyReachesSteadyState(
+                test_client, "UNARY_CALL", _INITIAL_UNARY_MAX_REQUESTS, 1
+            )
+            self.assertClientEventuallyReachesSteadyState(
+                test_client, "EMPTY_CALL", _INITIAL_EMPTY_MAX_REQUESTS, 1
+            )
+
+        with self.subTest("13_increase_backend_max_requests"):
+            self.td.backend_service_patch_backends(
+                circuit_breaker={"maxRequests": _UPDATED_UNARY_MAX_REQUESTS}
+            )
+
+        with self.subTest("14_client_reaches_increased_steady_state"):
+            self.assertClientEventuallyReachesSteadyState(
+                test_client, "UNARY_CALL", _UPDATED_UNARY_MAX_REQUESTS, 1
+            )
+
+
+if __name__ == "__main__":
+    absltest.main(failfast=True)
```

Comment:
This was kinda needed for security tests, because we didn't want to run the whole suite if anything fails, and this got copy-pasted everywhere.

```suggestion
    absltest.main()
```